src/app/frontend/common/components/actionbars/scaledefault/component.ts

```diff
@@ -49,4 +49,11 @@ export class ScaleDefaultActionbar implements OnInit, OnDestroy {
     this._unsubscribe.next();
     this._unsubscribe.complete();
   }
+
+  scalable(): boolean {
+    if (this.resourceMeta.typeMeta.scalable !== undefined) {
```

Comment:
Why return true if scalable is undefined?

Reply:
It has to do with [this](https://github.com/kubernetes/dashboard/pull/4195#discussion_r317648007). For deployments and other scalable resources, we're not yet setting this field from the backend.

Reply:
Add comment with some todo here that backend should be adapted and this should be changed.

I'd also change it to `return this.resourceMeta.typeMeta.scalable === undefined || this.resourceMeta.typeMeta.scalable`.

Reply:
Done :+1:

Reply:
I've moved the `scalableResources` implementation to the backend for other resources.